srcDLL/op2ext.cpp

```diff
@@ -104,3 +103,15 @@ OP2EXT_API bool IsConsoleModuleLoaded(const char* moduleName)
 {
 	return consoleModLoader.IsModuleLoaded(moduleName);
 }
+
+OP2EXT_API bool IsIniModuleLoaded(const char* moduleName)
+{
+	return iniModuleLoader.IsModuleLoaded(moduleName);
+}
+
+OP2EXT_API size_t GetLoadedModuleCount()
+{
+	const std::size_t consoleModuleCount = (consoleModLoader.IsModuleLoaded() ? 1 : 0);
```

Comment:
Was not sure if parenthesis are needed for ternary operator use here. I have tended to avoid the ternary operator in the past since it can be confusing. Let me know if it would be better to separate into an if statement.

Reply:
You can eliminate the parenthesis. Ternary can be confusing to read, though I often prefer the more compact ternary form to multiline `if`/`else` blocks.